pyrsa/rdm/calc.py

```diff
@@ -464,6 +464,7 @@ def _gen_default_cv_descriptor(dataset, descriptor):
 
 
 def _calc_pairwise_differences(measurements):
+    measurements = measurements.squeeze()
```

Comment:
If measurements come from DatasetTime, measurements is three-dimensional. The last dimension is empty here.
It may be worth checking whether this causes trouble if either n_obs or n_channels are one.